compiler/src/typed/env.re

```diff
@@ -1870,7 +1874,11 @@ and store_extension = (~check, id, ext, env) => {
       switch (cstr.cstr_args) {
       | [] => ReprValue(WasmI32)
       | args =>
-        ReprFunction(List.map(_ => WasmI32, args), [WasmI32], Unknown)
+        ReprFunction(
+          List.map(_ => WasmI32, args),
+          [WasmI32],
+          Direct(Ident.unique_name(id)),
```

Comment:
Didn't poke around the code, but do we still use `Unknown` in other parts of the code?

Reply:
Yeah, it's still used.